sqrl-examples/udf/schema.graphqls

```diff
@@ -0,0 +1,20 @@
+type Query {
```

Comment:
I like the naming convention of giving the sqrl script and the schema the same name. Makes it easier to see what goes together. (i.e. myudf.graphqls)